XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectPatchPage.cs

```diff
@@ -150,12 +151,14 @@ public override void PageLoaded(PageLoadedDirection direction)
                 labelWithAutomatedUpdates.Visible = automatedUpdatesOptionLabel.Visible = AutomatedUpdatesRadioButton.Visible = automatedUpdatesPossible;
                 labelWithoutAutomatedUpdates.Visible = !automatedUpdatesPossible;
 
-                AutomatedUpdatesRadioButton.Checked = automatedUpdatesPossible;
-                downloadUpdateRadioButton.Checked = !automatedUpdatesPossible;
+                AutomatedUpdatesRadioButton.Checked = firstLoad && automatedUpdatesPossible;

```

Comment:
This wil set Checked to false if `firstLoad` is false, which should not happen. We should only set Checked programatically on the firstLoad (the default values) and maybe when the AutomatedUpdatesRadioButton gets hidden while navigating through the pages (yhis might happen if some hosts get disconnected)